src/components/ToDoItem.tsx

```diff
@@ -19,33 +19,41 @@ export const ToDoItem: React.FC<ToDoItem> = ({
   newTextWithId,
 }) => {
   const isEdit = newTextWithId?.id === item.id;
+  
+  const onInputChangeNewTextWithId = React.useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) =>
+      setNewTextWithId({
+        id: newTextWithId?.id ?? 0,
+        text: e.target.value,
```

Comment:
`text: e.target.value ?? ''`, or something like this, also this could be redundant, considering the code below